src/kernels/conv3x3wrw.s

```diff
@@ -77,6 +77,21 @@ default reverse_inout, 0
 default weights_layout, 0
 default reverse_weights, 0
 
+// gfx90a requires 64bit aligned vgpr tuples
+// Tuples are used only in buffer_load_dwordx/buffer_store_dwordx instructions
+//
+// To meet this requirement, the following approach is used ('buffer_load_dwordx4 v[x:y]' as an example):
+//    if 'x' 64bit aligned:
+//       buffer_load_dwordx4 v[x:y], ...
+//    if 'x' not 64bit aligned:
+//       buffer_load_dword   v[x], ...
+//       buffer_load_dwordx3 v[x+1:y], ...
+.if (.amdgcn.gfx_generation_number == 9 && .amdgcn.gfx_generation_stepping == 10)
```

Comment:
[Recommendation] This enables alignment for `9*a`, shall we better enable it for `90a`?